src/Controllers/AccountController.cs

```diff
@@ -0,0 +1,94 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Authorization;
+using TrainingNet.Models;
+using TrainingNet.Models.Views;
+
+
+namespace TrainingNet.Controllers
+{
+  [Route("[controller]"), Authorize]
+    public class AccountController : Controller
+    {
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        public SignInManager<ApplicationUser> SignInManager
+        {
+            get { return _signInManager; }
+        }
```

Comment:
Add an empty line to separate the constructors

Reply:
ok